hosting_docs/source/installation/2-manual-install.rst

```diff
@@ -175,6 +175,45 @@ our example cluster, the control machine is named “control1”.
 
         $ sudo -H pip install ansible virtualenv virtualenvwrapper --ignore-installed six
 
+Install Python 3.10 (optional)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Python 3.10 requires additional setup on Ubuntu 18.04 to work properly.
+
+1.  Install system packages
+
+    ::
+
+        $ sudo add-apt-repository -y ppa:deadsnakes/ppa
+        $ sudo apt update
+        $ sudo apt-get -y install python3.10 python3.10-dev python3.10-distutils libffi-dev
+
+2.  Install updated version of pip for use with python 3.10
+
+    ::
+
+        $ sudo python3 -m pip install --target=/usr/local/lib/python3.10/dist-packages pip setuptools
+        $ sudo python3.10 -m pip --upgrade pip
+
+    To successfully use pip with 3.10, we must use the system python/pip to install an updated pip in the 3.10 path.
+
+3.  Install necessary packages to run init.sh
+
+    ::
+
+        $ sudo python3.10 -m pip install virtualenv virtualenvwrapper
```

Comment:
Why is this necessary? [`init.sh` installs or upgrades virtualenv](https://github.com/dimagi/commcare-cloud/blob/master/control/init.sh#L25) in the users home directory. I think I prefer that approach to managing a centrally managed virtualenv that can only be upgraded by a superuser.

Why is virtualenvwrapper needed? I think we should not use it on control machines because it's more complicated than virtualenv, and we can automate everything we need with virtualenv.

Reply:
Ah good point. I was testing creating a virtualenv before running the script so I just kept this instruction in place without much thought. Will definitely remove this step.

Reply:
See 17f0aad0ea3c8f7e6cab14d24efaffca312635dd